test/inspector/inspector-helper.js

```diff
@@ -497,11 +496,12 @@ Harness.prototype.kill = function() {
 
 exports.startNodeForInspectorTest = function(callback,
                                              inspectorFlags = ['--inspect-brk'],
-                                             opt_script_contents) {
+                                             mainScript = 'loop.js') {
   const args = [].concat(inspectorFlags);
-  if (opt_script_contents) {
+  if (!mainScript.endsWith('.js')) {
```

Comment:
Do you have a guarantee that `typeof mainScript.endsWith === 'function'`?
Maybe try `!String.prototype.endsWith(mainScript, '.js')`
Also flip if/else and remove `!`